templates/_asset-card-image.html

```diff
@@ -60,14 +60,15 @@
         {% endfor %}
       </div>
       {% if details %}
+        <hr class="p-rule" />
+        <p>
+          Salesforce Campaigns: <ul class="p-list--divided">{% for campaign in asset.salesforce_campaigns %}<li  class="p-list__item">{{ campaign.name }}</li>{% if not loop.last %} {% endif %}{% endfor %}</ul>
```

Comment:
It isn't clear to me what the purpose of this is: `{% if not loop.last %} {% endif %}`

Reply:
This one was to add comma, but I realised later that the names are too big to be in single line, it seems to be artifact of that realisation.
Removed it.